proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -731,9 +728,10 @@ lemma replyRemoveTCB_corres:
     apply (rule corres_symb_exec_r_sr)
        apply (rule corres_guard_imp)
          apply (rule reply_unlink_tcb_corres[simplified dc_def])
-         apply (rule disjI2, simp)
-        apply (clarsimp dest!: valid_objs_valid_tcbs st_tcb_reply_state_refs
-                         simp: obj_at_def is_reply reply_tcb_reply_at_def)
+        apply (clarsimp dest!: valid_objs_valid_tcbs)
+        apply (intro conjI)
```

Comment:
Can you solve this and the next two lines with just one command? It might be nicer than using `intro conjI` and then relying on the order being as it is now